runtime/legion.h

```diff
@@ -2524,6 +2524,7 @@ namespace Legion {
       Task(void);
     public:
       virtual const char* get_task_name(void) const = 0;
+      virtual bool has_trace(void) const = 0;
```

Comment:
Is there a reason that we added this to the 'Task' object? I feel like this is something that should probably be in the mapping interface arguments and not something directly exposed in the 'Task' interface.